tools/Documentation/plugin-docs/login.md

```diff
@@ -14,12 +14,11 @@ When executing your Plugin, LRR will call the `do_login` subroutine and give it
 ```perl
 sub do_login {
 
-    #First lines you should have in the subroutine
-    shift;
-    my ($param1, $param2, $param3) = @_; # Plugin parameters
+    #First line you should have in the subroutine
+    my (undef, $params) = @_; # Plugin parameters
```

Comment:
Same here, I think the `shift;` way is easier to understand and preps contributors if they ever fancy working on the full codebase which uses this pattern everywhere.  

(I guess one might argue that `shift` is a bit of a magic word either when you don't know that perl subs pass the module reference as the first argument and that shift just eats it, but it is how it is..)